org.speed.dreams.yaml

```diff
@@ -0,0 +1,166 @@
+app-id: org.speed.dreams
+runtime: org.freedesktop.Platform
+runtime-version: '1.6'
```

Comment:
18.08 is preferred, if possible.

Reply:
Wath is the diferences bettenw 1.6 and 18.08?

Reply:
18.08 has a completely new base, makes 1.6 obsolete. And also it’s not good style to use older runtimes for no reason, because the package will just get out-of-date sooner and new runtimes (should) = better runtimes, since there are a lot of updates in it.